ui/widget.js

```diff
@@ -246,7 +257,7 @@ $.widget.bridge = function( name, object ) {
 	};
 };
 
-$.Widget = function( /* options, element */ ) {};
+$.Widget = function( /* Options, element */ ) {};
```

Comment:
@arschmitz
>  ( and did before landing )

I wounder, did you aware that not add spasing in parentheses in English now? :-)

Yeah, `requireCapitalizedComments` requires some work - https://github.com/jscs-dev/node-jscs/issues/1686 could you add cases you find in there?

>  I was trying to get 100% passage with out having to have this rule exceptions from the jQuery preset

I guess if rule mess things up instead of helping, we probably should consider removing it from the preset until its working right